.gitignore

```diff
@@ -2,4 +2,5 @@ build
 phpunit.xml
 composer.phar
 composer.lock
-vendor
\ No newline at end of file
+vendor
+nbproject/
```

Comment:
Not a big issue, but editor excludes should be in your global gitignore so the project only has project specific ignores.